boot-qemu.sh

```diff
@@ -90,6 +94,11 @@ function sanity_check() {
     [[ -z ${ARCH} ]] && die "Architecture ('-a') is required but not specified!"
     [[ -z ${KERNEL_LOCATION} ]] && die "Kernel image or kernel build folder ('-k') is required but not specified!"
 
+    # --debian requires --interactive
+    [[ -z ${DEBIAN} ]] && DEBIAN=false
+    [[ -z ${INTERACTIVE} ]] && INTERACTIVE=false
+    ${DEBIAN} && ! ${INTERACTIVE} && die "'--debian' requires '-i'/'--interactive'!"
```

Comment:
how about the `--debian` flag just sets the `-i` flag implicitly?

Reply:
Good catch, I forgot that we do that with `-g`/`--gdb`.